frontend/lib/tools/auto.ts

```diff
@@ -30,6 +31,21 @@ export async function autoAnswer(
         const newMessages = getHistoryMessages(isPro, messages);
         const query = newMessages[newMessages.length - 1].content;
 
+        let summaryTitle = '';
+        let summaryText = '';
+
+        try {
+            const { text } = await generateText({
```

Comment:
No need for await, we should prioritize streamText, and we can await generateText after streamText ends.